setup.py

```diff
@@ -26,6 +26,8 @@
     'rsa>=3.1.4,<5; python_version >= "3"',
     "setuptools>=40.3.0",
     "six>=1.9.0",
+    "aiohttp>=3.6.2; python_version>='3.6'",
+    "aioresponses>=0.6.4; python_version>='3.6'",
```

Comment:
is aioresponses needed for test and client code?

Reply:
Resolved above comments. aioresponses is used as a mocking framework for async io responses, so is only required for test code. Changed to only include in the test dependencies,

Reply:
Yes, the key_path file is generated when nox is run (even after deleting the file), so it does seem to be required.

Reply:
This instance member is separate to the sync version, the reason i closed it here is to prevent the issue of hanging sessions.